convert2rhel.py

```diff
@@ -41,41 +66,83 @@ def create_report(reportfile, log_destination_location):
     where the rhc-worker-script parse the result of the c2r run
     """
     create_log_folder()
-    cmd = ["cp", reportfile, log_destination_location]
-    process = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.STDOUT, bufsize=1)
-    output = ""
-    for line in iter(process.stdout.readline, b""):
-        line = line.decode("utf8")
-        output += line
-
-    # Wait for the process to end
-    process.wait()
-
-    if process.returncode != 0:
-        print(output)
+    try:
+        with open(log_destination_location, "w") as f:
+            f.write(json.dumps(reportfile))
+    except:
         raise RuntimeError("Mock failed to create a report")
 
 
+'''
+    Crafts a report adding all entries from :path_issues to the
+    base reportfile located in :reportfile_path
+
+    :param reportfile_path: Path to the base reportfile
+    :param report_issues: Dict with the paths to the reportfiles to add
+                        Dict has to be str -> str.
+'''
+def craft_report(reportfile_path, report_issues):
+    BASE_REPORT_CRAFT_FOLDER = BASE_REPORT_DATA_FOLDER + "craft/"
+
+    reportfile = {}
+
+    # Get the base reportfile
+    with open(reportfile_path, mode="r") as f:
+        reportfile = json.load(f)
+
+    # Push all report_issues into base reportfile
+    for report_issue in report_issues.values():
+        REPORT_CRAFT_FOLDER = BASE_REPORT_CRAFT_FOLDER + report_issue
```

Comment:
I suggest that `report_issue` value would already contain the `.json` postfix. I don't see it being used anywhere elese without the json postfix and it would be more explaining that you are refferencing to the file when you are populating the dictionary.

Reply:
I did this change and realised I wasn't using the keys from the dictionary. I updated report_issues to an array and added the `.json` to the original appended value.

Reply:
Array looks better now, thanks!